dotnet/src/SemanticKernel.Abstractions/Orchestration/SKContext.cs

```diff
@@ -74,25 +61,6 @@ public SKContext Clone(ContextVariables? variables)
         return new SKContext(
             variables ?? this.Variables.Clone(),
             this.FunctionInvokingHandler,
-            this.FunctionInvokedHandler,
-            this.Culture);
-    }
-
-    /// <summary>
-    /// The culture currently associated with this context.
-    /// </summary>
-    private CultureInfo _culture;
-
-    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-    private string DebuggerDisplay
```

Comment:
If you're deleting DebuggerDisplay, you should also delete the `[DebuggerDisplay("{DebuggerDisplay,nq}")]` on the class.

Reply:
fixed